eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -47,6 +50,33 @@ $headers = @{
 
 $query = "state=open&head=${PROwner}:${PRBranch}&base=${BaseBranch}"
 
+function AddLabels([int] $prNumber)
+{
+  # Adding labels to the pr.
+  if (-not $PRLabel) {
+    Write-Verbose "There are no labels added to the PR."
+    exit 0
```

Comment:
We shouldn't call exit from a function as it will exit the entire script.

Reply:
I suggest just returning if the labels are empty.

Reply:
Return if empty